scalafmt-tests/src/test/resources/newlines/source_fold.stat

```diff
@@ -4623,9 +4673,8 @@ object a {
     s"Cannot parse parameter f as DummyEnum, $wrongValue is not a member of Enum (dummy-value)"
   ))
 
-  when(audienceService.publish(any[Tenant], any[Int]))
-    .thenReturn(Success[NonEmptyList[String], ApiAudience](apiAudience
-      .copy(status = AudienceStatus.Pending.name)))
+  when(audienceService.publish(any[Tenant], any[Int])).thenReturn(Success[NonEmptyList[String],
```

Comment:
Is this an expected change?

Reply:
the previous formatting wasn't because `audience.copy` was formatted multiline. but let's check: @sjrd , does the rule apply when you have only one argument?

Reply:
Yes, the change seems correct and expected. It does apply when you have only one argument.

By hand I would probably try to nudge scalafmt into formatting it as
```scala
  when(audienceService.publish(any[Tenant], any[Int]))
    .thenReturn(Success[NonEmptyList[String], ApiAudience](
        apiAudience.copy(status = AudienceStatus.Pending.name)))
```
to avoid breaking in the middle of a *type* parameter list. But that's not a rule we really have, because we never needed it, because our codebases don't have long types like this. Given a setting with hand-formatting, we came up with whatever looked good in the rare instances where it appeared.

So as far as I am concerned, I'm happy with the green version.

Reply:
Looks good then!